Enhancer/ContentRepositoryEnhancer.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+/*
+ * This file is part of the Symfony CMF package.
+ *
+ * (c) 2011-2015 Symfony CMF
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Cmf\Component\Routing\Enhancer;
+
+use Symfony\Cmf\Component\Routing\ContentRepositoryInterface;
+use Symfony\Cmf\Component\Routing\RouteObjectInterface;
+use Symfony\Component\HttpFoundation\Request;
+
+class ContentRepositoryEnhancer implements RouteEnhancerInterface
+{
+    /**
+     * @var ContentRepositoryInterface
+     */
+    private $contentRepository;
+
+    /**
+     * @var string
+     */
+    private $target;
+
+    /**
+     * @var string
+     */
+    private $source;
+
+    /**
+     * ContentRepositoryEnhancer constructor.
+     *
+     * @param ContentRepositoryInterface $contentRepository
+     * @param string                     $target
+     * @param string                     $source
```

Comment:
please add a short description that these are the field names and that the source is used as id on the content repository and target is the field where its written to